lib/use_cases/save_items_details.rb

```diff
@@ -5,17 +5,19 @@ def initialize(items_gateway:)
     @items_gateway = items_gateway
   end
 
-  def execute(items_details:)
-    @items_details = items_details
+  def execute(items:)
+    @items = items
 
     errors = validation
 
     return { successful: false, errors: errors } unless errors.empty?
+    
+    item_builder = ItemBuilder::Item.new
+    item_builder.items = @items
```

Comment:
The naming is a bit confusing. could the item method be called order?
`item_builder.order = @items`
Could the builder be called order builder?

Reply:
Yes, when there is an Order domain object that holds an array of items.

Reply:
Agree with Jaseera. This code is fine for now until we implement an Order domain object that takes the items.